cmd/server.go

```diff
@@ -0,0 +1,75 @@
+/*
```

Comment:
can we move it to cmd/server/server.go and the package can then be main, reference project to layout cmds https://github.com/kubernetes/kubernetes/tree/master/cmd